Polyhedron/demo/Polyhedron/Plugins/PMP/Isotropic_remeshing_plugin.cpp

```diff
@@ -206,13 +206,16 @@ class Polyhedron_demo_isotropic_remeshing_plugin :
     || qobject_cast<Scene_polyhedron_selection_item*>(scene->item(scene->mainSelectionIndex()));
     }
 
+    bool ok(true), found_poly(false);
+
     Q_FOREACH(int index, scene->selectionIndices())
     {
-      //if one polyhedron is found in the selection, it's fine
-      if (qobject_cast<Scene_facegraph_item*>(scene->item(index)))
-        return true;
+      if (!qobject_cast<Scene_facegraph_item*>(scene->item(index)))
+        ok = false;
+      else
+        found_poly=true;
     }
-    return false;
+    return ok && found_poly;
```

Comment:
IMHO the case where the user selects a `Scene_face_graph_item` and its associated `Scene_polyhedron_selection_item` should return `true`

Reply:
it is counter intuitive, when you select both you expect only the selectio nto be touched, when it will actually be the exact opposite.

Reply:
When you select both, it should remesh the Surface_mesh, while taking into account the selection : selected facets should be remeshed, selected edges and vertices considered as constraints

Reply:
That's what happens when you select only the selection_item. When you select several items, it remeshes all the selected items.